impala/_rpc/beeswax.py

```diff
@@ -21,8 +21,9 @@
 from six.moves import range
 
 from impala.error import RPCError, QueryStateError, DisconnectedError
+from impala._rpc import thrift_util
```

Comment:
Nit: for consistent style, maybe change to `from impala._rpc.thrift_util import ...`